SageMaker/02_classification.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<p>I think a little description of what this estimator is and does would be helpful. Also not a big fan of tabbed kwargs, but no strong opinion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nlp-with-transformers/notebooks/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>